tests/scripts/setup_binary.sh

```diff
@@ -8,9 +9,16 @@ VAULT_DIR=$(cd `dirname $0` && pwd)
 BUILD_COMMAND=make
 OUTPUT_DIR=$(pwd)
 TEST_DIR=$CMAKE_SOURCE_DIR/test_binary/tests
-EYRIE_DIR=$CMAKE_SOURCE_DIR/../rts/eyrie
+EYRIE_DIR=$CMAKE_BINARY_DIR/eyrie
 EYRIE_PLUGINS="freemem"
 
+# Download Eyrie Runtime
+if [ ! -d $EYRIE_DIR ]
+then
+  git clone https://github.com/keystone-enclave/keystone-runtime $EYRIE_DIR
+fi
+cd $EYRIE_DIR; git checkout 3130f20; cd ..
```

Comment:
Do we want to have a specific eyrie checkout hash here? (Do we expect setup_binary to work with any newer eyries?)

Reply:
This is for testing the measurement, so we do need a specific eyrie version so that we know the exact measurement.

Reply:
Hmm. The measurement will then depend on the exact version of all the included libs + eyrie + toolchain.
I'm not sure we can easily support a 'known measurement' test without it breaking all the time?

Reply:
Yeah, you're right that it will be broken whenever we change the edge lib or toolchain even if we fix the eyrie version.
I was hoping to upload the specific binary somewhere or to build a very very small fake binary (instead of the runtime) in the repo to do this particular unit test. 
I think that can go as a separate PR.